.github/workflows/jira-branch-check.yml

```diff
@@ -0,0 +1,23 @@
+name: Check Jira key in branch exists and is valid
```

Comment:
Does this check make sense for a public repo? Any outside contributors wouldn't have access the the private Jira board?

Reply:
We have those checks also for the other public projects (AdSecGH, ComposGH and GsaGH). I am happy to get rid off them though ;)